### ISSUE TITLE
'NoneType' object has no attribute 'encode'

### ISSUE THREAD

**Issue by polonezu1985** [ShayanJa/RL_trader#9]
File "C:\Users\ftudo\anaconda3\lib\site-packages\binance\client.py", line 175, in _generate_signature
    m = hmac.new(self.API_SECRET.encode('utf-8'), query_string.encode('utf-8'), hashlib.sha256)!!is making me crazzy this error ,please help

**Comment by ShayanJa** [comment]
You need to get a binance api key

**Comment by polonezu1985** [comment]
Yes ,i Have the api keys but the error stilul persist!maybe You can remote
Connect to my PC   en help  me ,that will be great !!

vin., 29 apr. 2022, 23:50 ShayanJa ***@***.***> a scris:
